src/ReadoutConstants.hpp

```diff
@@ -35,6 +35,9 @@ const constexpr std::size_t FLX_SUPERCHUNK_FACTOR = 12;
 
 const constexpr std::size_t WIB_SUPERCHUNK_SIZE = 5568; // for 12: 5568
 
+// TP 
+const constexpr std::size_t TP_SUPERCHUNK_SIZE = 5568; // may change
```

Comment:
With this, basically we align every TP message to 5568Bytes. I understand, that this is temporary, but we should push for the dynamic output sink.